### PR TITLE
Update create config + 1.0b5 updates

### PR DESCRIPTION
**Pull request by maaikelimper**
New PR with the branch on my test-server, which combines the updates for update-create-config and the changes for 1.0b5 (wis2box-webapp and new wis2box-api from branch).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -40,6 +40,8 @@ nosetests.xml
 wis2box-data
 .pytest_cache
 dev.env
+wis2box.env
 docker/.env
 
 .ipynb_checkpoints
+wis2box.env
```
Comment (david-i-berry) [inline .gitignore:47]: duplicate?

File: docker-compose.dev.yml [hunk 2]
```diff
@@ -4,7 +4,7 @@ services:
       - ${WIS2BOX_HOST_DATADIR}:/data/wis2box:rw
       - ./wis2box-management/wis2box/wis2box.cron:/etc/cron.d/wis2box:ro
       - ./wis2box-management/wis2box:/app/wis2box
-    command: ["wis2box", "pubsub" , "subscribe", "--broker", "http://wis2box-minio:9000", "--topic", "wis2box-storage/#", "--verbosity", "INFO"]
+    command: ["wis2box", "pubsub" , "subscribe"]
```
Comment (david-i-berry) [inline docker-compose.dev.yml:7]: Options no longer required?

File: docs/source/community/troubleshooting.rst [hunk 3]
```diff
@@ -86,7 +85,7 @@ ERROR - Failed to publish, wsi: ..., tsi: XXXXX
 -----------------------------------------------
 
 Data arrived for a station that is not present in the station metadata cache. 
-To add missing stations, update the file ``metadata/station/station_list.csv`` in the wis2box data directory (see :ref:`setup`).
+To add missing stations, use the station-editor in wis2box-webapp (from wis2box-1.0b5) or update the file ``metadata/station/station_list.csv`` in the wis2box data directory (see :ref:`setup`).
```
Comment (david-i-berry) [inline docs/source/community/troubleshooting.rst:88]: Does updating station_list still update the station collection? I know we had a discussion on this but can't remember what the outcome was.

File: docs/source/reference/auth.rst [hunk 4]
```diff
@@ -16,11 +16,19 @@ To add a token to the execution of a wis2box process, use the following command:
 
 .. code-block:: bash
 
-    wis2box auth add-token --path processes/wis2box myexecutiontoken
+    wis2box auth add-token --path processes/wis2box
```
Comment (david-i-berry) [inline docs/source/reference/auth.rst:19]: Do we want to add that they can specify a token as well?

File: wis2box-management/wis2box/metadata/station.py [hunk 5]
```diff
@@ -345,12 +404,12 @@ def get(ctx, wsi, verbosity):
     results = OrderedDict({
         'station_name': station['station_name'],
         'wigos_station_identifier': station['wigos_station_identifier'],
```
Comment (david-i-berry) [inline wis2box-management/wis2box/metadata/station.py:406]: get? also for the other keys?

File: docker-compose.dev.yml (hunk 2 above)
Comment (maaikelimper) [reply docker-compose.dev.yml:7]: the options were mostly not used inside subscribe.py . The topic used for subscription can not be a configurable at this time as it set in other places (such as acl.conf for mosquitto, notification-topic for minio) and subscribing to any other topic than 'wis2box/#' would breake the wis2box-workflow.

File: docs/source/community/troubleshooting.rst (hunk 3 above)
Comment (maaikelimper) [reply docs/source/community/troubleshooting.rst:88]: You can still use the station_list.csv to do a bulk-insert the station collection, by executing the command 'wis2box metadata station publish-collection', as documented in setup.rst.

File: docs/source/reference/auth.rst (hunk 4 above)
Comment (maaikelimper) [reply docs/source/reference/auth.rst:19]: Yes I can add this to the docs

**Comment by maaikelimper** [comment]
Comment to self: add additional sleep before running integration tests in GHA: to help avoid intermittent CI failures

File: wis2box-management/wis2box/metadata/station.py (hunk 5 above)
Comment (maaikelimper) [reply wis2box-management/wis2box/metadata/station.py:406]: I did not write this part of the code, but I believe the station-report from pyoscar always returns the same attributes

**Comment by maaikelimper** [comment]
I noted the cronjob for cleaning data was broken and added a comment to use the correct environment variable. I increased the default retention-period for data from 7 to 30.

**Review by tomkralidis — APPROVED** [review]

**Pull request merged**